app/views/admin/reports/_rendering_options.html.haml

```diff
@@ -14,6 +14,10 @@
     .row
       .alpha.two.columns= label_tag nil, t(".display")
       .omega.fourteen.columns
+        - if @report.metadata_option?
+          %span.inline-checkbox
+            = check_box_tag :include_metadata, true, ActiveModel::Type::Boolean.new.cast(@rendering_options.options[:include_metadata]), id: 'include_metadata'
+            = label_tag :include_metadata, t('.include_metadata')
```

Comment:
It's a bit confusing if you check this box but then don't choose CSV, you won't get any metadata.